testcafe/tests/pages/drive/drive-modal-model.js

```diff
@@ -0,0 +1,31 @@
+import {
+  getElementWithTestId,
+  isExistingAndVisibile
+} from '../../helpers/utils'
+import { t } from 'testcafe'
+import DrivePage from './drive-model'
+
+export default class MoveModal extends DrivePage {
+  constructor() {
+    super()
+
+    //Overwrite some selector to use our regular drive method in modal context
+    this.modaleContent = getElementWithTestId('fil-content-modal')
+
+    this.breadcrumb = this.modal
+      .find('h2')
+      .withAttribute('data-test-id', 'path-title')
+    this.folderOrFileName = this.modaleContent
+      .find('div')
+      .withAttribute('data-test-id', 'fil-file-filename-and-ext')
+  }
+
+  async moveElementTo(destinationFolder) {
+    await this.goToBaseFolder()
+    await this.goToFolder(destinationFolder)
```

Comment:
I'm confused: `goToFolder ` is used to navigate to another folder on the main page, not on the modal. How does it know to navigate inside the modal?

Reply:
Because the selector use to `goToFolder` , `this.folderOrFileName` is redefined here, so I can use the same function. But this will likely change while working on https://trello.com/c/22GedOwK/1850-%F0%9D%90%92-check-de-tous-les-selecteurs-sortir-des-classes-et-faire-un-fichier-communs-avec-tous-les-selecteurs-en-exports

Reply:
Mmh ok. This is definitely a bit of a tricky behavior, so we'll see how we can improve it with the other card then :)